src/main/kotlin/com/matthewrussell/trchunkbrowser/app/mainview/MainView.kt

```diff
@@ -214,7 +214,8 @@ class MainView : View() {
                                     "${segment.label.padStart(2, '0')}") {
                                 addClass(MainViewStyles.segmentTitle)
                             }
-                            val takeInfo = segment.src.nameWithoutExtension.split("_").filter { it.startsWith("t") }
+                            val takeInfo = segment.src.nameWithoutExtension.split("_")
+                                .filter { it.matches("t\\d+$".toRegex()) }
                             if(takeInfo.isNotEmpty()) {
                                 val take = takeInfo.last()
```

Comment:
This approach to getting the take number seems clunky, but since I'm not familiar with what the file names look like, I'm not sure if it can be accomplished with a single regex?

Reply:
@KJoslyn Like this for example? Or is it even more clunky? 
```
val takeInfo = "t(\\d+)$".toRegex()
    .find(segment.src.nameWithoutExtension)
    ?.groupValues?.get(1)

if(takeInfo !== null) {
    label("Take $takeInfo") {
        addClass(MainViewStyles.segmentInfo)
    }
}
```

Reply:
Look good!